Plugins/GitUIPluginInterfaces/GitRevision.cs

```diff
@@ -89,6 +89,7 @@ private static DateTime FromUnixTimeSeconds(long unixTime)
 
         public string Subject { get; set; } = "";
 
+        /// <summary> Full commit message (all lines), but without Notes. </summary>
```

Comment:
Please use multiline (i.e., tags on their own lines), and use `<see cref="Notes"/>`.